docs/blog/arcade/intro-cloud-sync.md

```diff
@@ -0,0 +1,40 @@
+# Introducing Cloud Sync for MakeCode Arcade
+
+**Posted on August 12th, 2021 by [eanders-ms](https://github.com/eanders-ms)**
+
+**Note**: As of this writing, sign-in and cloud sync are only available in [MakeCode Arcade](https://arcade.makecode.com).
+
+Over the past year we've been working to make it possible to sign into MakeCode with a [Microsoft Account](https://aka.ms/AAdd6f8). When you sign into MakeCode, you make it possible for us to remember who you are, and this unlocks a bunch of exciting new possibilities. For example, it becomes possible for us to remember what you were working on and make your projects available wherever you are. In fact this is the first feature we've built for our signed in users, and we're calling it **Cloud Sync**.
+
```

Comment:
Change this to match the language above

I would also change it to either:
- "make your projects available wherever you are on any device" or "make your projects available on any device you use" (i just think the any device should be emphasized)

Reply:
How about "wherever you use MakeCode"?